projects/mpmath.org/package.yml

```diff
@@ -0,0 +1,24 @@
+distributable:
+  url: https://mpmath.org/files/mpmath-{{version}}.tar.gz
+  strip-components: 1
+
+versions:
+  url: https://mpmath.org
+  match: /mpmath-\d+\.\d+\.\d+\.tar\.gz/
+  strip:
+    - /^mpmath-/
+    - /\.tar\.gz$/
+
+dependencies:
+  python.org: '>=3.11'
+
+runtime:
+  env:
+    PYTHONPATH: ${{prefix}}/lib/python{{deps.python.org.version.major}}/site-packages:$PYTHONPATH
+
+build:
+  - python -m pip install --prefix={{prefix}} .
+  - run: ln -s python{{deps.python.org.version.marketing}} python{{deps.python.org.version.major}}
```

Comment:
this seems foolish unless mpmath have an explicit guarantee in their documentation that this will work.

If so then it is absolutely needed that we add a comment for that above this line.